src/pages/projects/the-beginner-s-guide-to-vue-3/index.tsx

```diff
@@ -0,0 +1,230 @@
+  const {title, ogImage, path, image, tags} = course
+
+  const introduction: any = find(course?.projects?.content, {
+    label: 'Introduction',
+  })
+  const projectBrief: any = find(course?.projects?.content, {
+    label: 'Project Brief',
+  })
+  const appRequirements: any = find(course?.projects?.content, {
+    label: 'App Requirements',
+  })
+  const appData: any = find(course?.projects?.content, {
+    label: 'App Data',
+  })
+  const developmentStandards: any = find(course?.projects?.content, {
+    label: 'Development Standards',
+  })
+  const stretchGoal: any = find(course?.projects?.content, {
+    label: 'Stretch Goal',
+  })
+  const appDesign: any = find(course?.projects?.content, {
+    label: 'App Design',
+  })
+  const Submission: any = find(course?.projects?.content, {label: 'Submission'})
+
+  const {productCard, productPage} = course?.projects
+
+  console.log(course)
+
+  return (
+    <>
+      <NextSeo
+        description={course.description}
+        title={title}
+        titleTemplate={'%s | egghead.io'}
+        twitter={{
+          site: `@eggheadio`,
+          cardType: 'summary_large_image',
+        }}
+        openGraph={{
+          title,
+          description: course.description,
+          site_name: 'egghead',
+          images: [
+            {
+              url: ogImage,
+            },
+          ],
+        }}
+      />
+      <div className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
+        <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+          <div className="mt-10 mb-16 text-center">
+            <div className="mb-10">
+              <Image priority src={image} height="270" width="270" />
+            </div>
+            <p className="text-lg md:text-2xl leading-6 text-gray-500">
+              Portfolio Project
+            </p>
+            <h1 className="text-2xl md:text-4xl font-bold mt-2">
+              {course.projects.title}
+            </h1>
+          </div>
+          <div className="flex flex-col justify-center items-start mx-auto max-w-screen-md mb-16 w-full px-4">
+            <main className="prose dark:prose-dark prose-lg max-w-none w-full">
+              <Markdown>{introduction.text}</Markdown>
+              <Markdown>{projectBrief.text}</Markdown>
+              <section className="grid grid-cols-2 gap-4">
+                <Markdown className="col-span-1">
+                  {appRequirements.text}
+                </Markdown>
+                <Markdown className="rounded mt-20">{appData.text}</Markdown>
+              </section>
+              <Markdown>{developmentStandards.text}</Markdown>
+
+              <ProjectCTA
+                title="Show you're hard won Vue skills. Make it stick."
+                meta="Fork the codesandbox and get started with your solution."
+                buttonText="Start the Vue Fundamentals Project"
+                projectLink={course.projects.projectLink}
+              />
```

Comment:
Blue text is mostly used for links.  "Fork the codesandbox and get started with your solution." is a secondary text. Although we are using the same component in Own Your Online Presence, I would suggest changing the text to use a secondary color. 

![image](https://user-images.githubusercontent.com/57044804/120007871-3d068c80-bf8f-11eb-8231-1d36c68e6107.png)

`codesandbox` -> `CodeSandbox`

Consider using multiple font sizes pending on the breakpoint: 
![image](https://user-images.githubusercontent.com/57044804/120008054-6de6c180-bf8f-11eb-9714-26d92eb93c7b.png)